src/guillo-gmi/reducers/guillotina.js

```diff
@@ -49,7 +49,11 @@ export function guillotinaReducer(state, action) {
       return { ...state, action: { action: undefined, params: undefined } }
 
     case 'REFRESH':
-      return { ...state, refresh: Math.random(), loading: true }
+      return {
+        ...state,
+        refresh: Date.now(),
```

Comment:
No matter how small the probability is, there is the probability that the value of `Math.random()` is repeated and does not refresh... Replacing to `Date.now()` fix this.